addons/mail/static/src/models/emoji_list_view/emoji_list_view.js

```diff
@@ -16,10 +16,10 @@ registerModel({
          * @param {MouseEvent} ev 
          */
         onClickEmoji(ev) {
-            if (this.popoverViewOwner.messageActionListOwnerAsReaction) {
+            if (this.popoverViewOwner && this.popoverViewOwner.messageActionListOwnerAsReaction) {
                 this.popoverViewOwner.messageActionListOwnerAsReaction.onClickReaction(ev);
             }
-            if (this.popoverViewOwner.composerViewOwnerAsEmoji) {
+            if (this.popoverViewOwner && this.popoverViewOwner.composerViewOwnerAsEmoji) {
```

Comment:
Prefer `} else if {` for the second conditional branching. You should no longer need guard